frontend/device/cervantes/device.lua

```diff
@@ -41,6 +41,8 @@ local Cervantes = Generic:new{
     hasOTAUpdates = yes,
     hasKeys = yes,
     hasWifiManager = yes,
+    canPowerOff = yes,
```

Comment:
and here (and in the other device files)

Reply:
Done.